antevus-app/fix_final_eslint.py

```
@@ -0,0 +1,98 @@
+# Remove old config
+if os.path.exists('.eslintrc.json'):
+    os.remove('.eslintrc.json')
+    print("Removed .eslintrc.json")
+
+# Remove old eslintignore
+if os.path.exists('.eslintignore'):
+    os.remove('.eslintignore')
+    print("Removed .eslintignore")
+
+print("\nRunning ESLint to check remaining issues...")
+result = subprocess.run(['npm', 'run', 'lint'], capture_output=True, text=True)
```

Ruff (0.13.1)

Starting a process with a partial executable path

(S607)